### DISCUSSION TITLE
Applying profiles to object

### DISCUSSION THREAD

**Discussion by wjdingman1** [ocsf/ocsf-schema#1080] [General]
Quick question here

I have a profile that I have developed which I have applied to a device. However, when I use the API Server to get the json schema for an `inventory_info` event with the profile as a query param, I fail to see the profile applied to the nested device object.

Is this because only the device is using the profile? How would you suggest a way around this?

Thanks!

**Comment by floydtree** [comment]
Hey apologies for such a delayed response. We seemed to have overlooked org-level discussion posts. I have now transferred it to be a repo level discussion where all of our discussions are. Have you been able to resolve the issue or are you still looking for help?
